.github/workflows/master.yml

```diff
@@ -0,0 +1,50 @@
+name: CI for master branch
+
+on:
+  push:
+    branches: master
+
+jobs:
+  push_images:
+    name: Push Docker images
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        image:
+          - fdb-kubernetes-operator
+          - foundationdb-kubernetes-sidecar
+          - fdb-data-loader
+        include:
+          - image: fdb-kubernetes-operator
+            context: ./
+          - image: foundationdb-kubernetes-sidecar
+            context: ./foundationdb-kubernetes-sidecar
+          - image: fdb-data-loader
+            context: ./sample-apps/data-loader
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      # Will be deprecated -> https://github.com/docker/login-action#github-packages-docker-registry
+      # but GitHub Container Registry is still in beta and only works with a personal access token (PAT)
+      # See: https://docs.github.com/en/packages/guides/migrating-to-github-container-registry-for-docker-images
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: docker.pkg.github.com
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Build and push to registry
+        uses: docker/build-push-action@v2
+        with:
+          context: ${{ matrix.context }}
+          tags: docker.pkg.github.com/${{ github.repository }}/${{ matrix.image }}:latest
+          # push: true
+          # Workaround: otherwise we have a multi-arch image which is not supported by the deprecated
+          # GitHub packages but GHCR doesn't support the workflow credentials and is only in public beta :(
+          # https://github.com/docker/build-push-action/blob/master/TROUBLESHOOTING.md#errors-on-pushing-to-registry
+          outputs: type=docker
+      # Really? -> https://github.community/t/docker-pull-from-public-github-package-registry-fail-with-no-basic-auth-credentials-error/16358/85
+      - name: Push the image with docker
+        run: docker push docker.pkg.github.com/${{ github.repository }}/${{ matrix.image }}:latest
```

Comment:
Does this mean that the images will be available under `docker.pkg.github.com` instead of `docker.com`, or is that a proxy?

Reply:
It will be published to the GitHub packages which will be deprecated by the GitHub container registry (which doesn't support the ` ${{ secrets.GITHUB_TOKEN }}` currently and is in beta. Also the GitHub packages doesn't allow anonymous pull (so we should either use DockerHub or GitHub container registry).

Reply:
If possible, I think we should push to DockerHub so that we don't disrupt any existing usage.